src/components/syncPlay/ui/settings/SettingsEditor.js

```diff
@@ -96,14 +96,22 @@ class SettingsEditor {
     async initEditor() {
         const { context } = this;
 
-        context.querySelector('#txtExtraTimeOffset').value = toFloat(getSetting('extraTimeOffset'), 0.0);
-        context.querySelector('#chkSyncCorrection').checked = toBoolean(getSetting('enableSyncCorrection'), true);
-        context.querySelector('#txtMinDelaySpeedToSync').value = toFloat(getSetting('minDelaySpeedToSync'), 60.0);
-        context.querySelector('#txtMaxDelaySpeedToSync').value = toFloat(getSetting('maxDelaySpeedToSync'), 3000.0);
-        context.querySelector('#txtSpeedToSyncDuration').value = toFloat(getSetting('speedToSyncDuration'), 1000.0);
-        context.querySelector('#txtMinDelaySkipToSync').value = toFloat(getSetting('minDelaySkipToSync'), 400.0);
-        context.querySelector('#chkSpeedToSync').checked = toBoolean(getSetting('useSpeedToSync'), true);
-        context.querySelector('#chkSkipToSync').checked = toBoolean(getSetting('useSkipToSync'), true);
+        context.querySelector('#txtExtraTimeOffset').value = toFloat(getSetting('extraTimeOffset'),
+            SyncPlay.Manager.playbackCore.extraTimeOffset);
+        context.querySelector('#chkSyncCorrection').checked = toBoolean(getSetting('enableSyncCorrection'),
+            SyncPlay.Manager.playbackCore.enableSyncCorrection);
+        context.querySelector('#txtMinDelaySpeedToSync').value = toFloat(getSetting('minDelaySpeedToSync'),
+            SyncPlay.Manager.playbackCore.minDelaySpeedToSync);
+        context.querySelector('#txtMaxDelaySpeedToSync').value = toFloat(getSetting('maxDelaySpeedToSync'),
+            SyncPlay.Manager.playbackCore.maxDelaySpeedToSync);
+        context.querySelector('#txtSpeedToSyncDuration').value = toFloat(getSetting('speedToSyncDuration'),
+            SyncPlay.Manager.playbackCore.speedToSyncDuration);
+        context.querySelector('#txtMinDelaySkipToSync').value = toFloat(getSetting('minDelaySkipToSync'),
+            SyncPlay.Manager.playbackCore.minDelaySkipToSync);
+        context.querySelector('#chkSpeedToSync').checked = toBoolean(getSetting('useSpeedToSync'),
+            SyncPlay.Manager.playbackCore.useSpeedToSync);
+        context.querySelector('#chkSkipToSync').checked = toBoolean(getSetting('useSkipToSync'),
+            SyncPlay.Manager.playbackCore.useSkipToSync);
```

Comment:
We can probably take the values directly because they are updated automatically.

The only thing that needs to be fixed (in other PR?) is that here should be an arrow function: https://github.com/jellyfin/jellyfin-web/blob/9778f2db657a3be6ec44173e618768873d186ff6/src/components/syncPlay/core/timeSync/TimeSyncCore.js#L51

_timeSyncCore fix from the above is included_
```suggestion
        context.querySelector('#txtExtraTimeOffset').value = SyncPlay.Manager.timeSyncCore.extraTimeOffset;
        context.querySelector('#chkSyncCorrection').checked = SyncPlay.Manager.playbackCore.enableSyncCorrection;
        context.querySelector('#txtMinDelaySpeedToSync').value = SyncPlay.Manager.playbackCore.minDelaySpeedToSync;
        context.querySelector('#txtMaxDelaySpeedToSync').value = SyncPlay.Manager.playbackCore.maxDelaySpeedToSync;
        context.querySelector('#txtSpeedToSyncDuration').value = SyncPlay.Manager.playbackCore.speedToSyncDuration;
        context.querySelector('#txtMinDelaySkipToSync').value = SyncPlay.Manager.playbackCore.minDelaySkipToSync;
        context.querySelector('#chkSpeedToSync').checked = SyncPlay.Manager.playbackCore.useSpeedToSync;
        context.querySelector('#chkSkipToSync').checked = SyncPlay.Manager.playbackCore.useSkipToSync;
```